test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/VersionRange.java

```diff
@@ -0,0 +1,31 @@
+package org.elasticsearch.test.rest.yaml.section;
+
+import org.elasticsearch.Version;
+
+public class VersionRange {
+    private final Version lower;
+    private final Version upper;
+
+    public VersionRange(Version lower, Version upper) {
+        this.lower = lower;
+        this.upper = upper;
+    }
+
+    public Version lower() {
+        return lower;
+    }
+
+    public Version upper() {
+        return upper;
+    }
+
+    public boolean contain(Version currentVersion) {
```

Comment:
...I don't suppose you'd rename this to `contains`? It's just that I'm so used to `Collection.contains` 😅

Reply:
it is a good idea, will do